.ci/README.md

```diff
@@ -0,0 +1,14 @@
+CI configuration for Elasticsearch
+==================================
+
+Layout and Syntax
+-----------------
+
+CI is run by Jenkins at [elasticsearch-ci](https://elasticsearch-ci.elastic.co/).
+Jobs live in the [jobs.t](jobs.t) directory, these are defined in YML using a syntax 
+simmilar to [JJB](https://elasticsearch-ci.elastic.co/view/Elasticsearch%20master/).
```

Comment:
What are the differences vs "vanilla" JJB? Is there some documentation in the infra repo we can link to here?

Reply:
There is but I didn't want to include it because it's not public.

Reply:
Would that be so bad? External folks would just get denied.